src/all_events.js

```diff
@@ -1,5 +1,18 @@
 let all_events = [
   {
+    "name": "UHack is Back!",
+    "color": "#f37423",
+    "date": {
+      "from": 1512236561000,
+      "to": 1512322961000
+    },
+    "event_url": "http://uhackis.life",
```

Comment:
Url seems to be dead right now. Is this the url for the event?

Reply:
I think it's good now. Let me know